lib/adapters/REST/endpoints/app-action.ts

```diff
@@ -16,6 +17,9 @@ const getBaseUrl = (params: GetAppDefinitionParams) =>
 const getAppActionUrl = (params: GetAppActionParams) =>
   `${getBaseUrl(params)}/${params.appActionId}`
 
+const getAppActionsEnvUrl = (params: GetSpaceEnvironmentParams) =>
+  `/spaces/${params.spaceId}/environments/${params.environmentId}/actions`
```

Comment:
FYI, in case they dont pass `envId`, it will still work (defaults to `master`). I would make it optional